Python/Product/Common/Strings.resx

```diff
@@ -2909,4 +2909,11 @@ Please specify at least one package.</value>
   <data name="WorkspaceText" xml:space="preserve">
     <value>workspace</value>
   </data>
+  <data name="UnitTestEnableInfoBarAction" xml:space="preserve">
+    <value>Enable unittest</value>
+  </data>
+  <data name="PythonTestFileDetected" xml:space="preserve">
+    <value>A file containing Python tests was detected in the {0} {1}, but the testing framework is disabled</value>
```

Comment:
Needs a '.' at the end of sentence.

The UI in project settings, and the property in msbuild and json file both refer to this as the "test framework", not "testing framework". I think we should keep it consistent and change the sentence to test framework.

"containing Python tests" is a little too confident given we didn't even look inside the file, but I guess that gets the point across.